### ISSUE TITLE
Discrepancy between lpass and webinterface

### ISSUE THREAD

**Issue by adrighem** [lastpass/lastpass-cli#335]
Hi,

The lpass tool sync seems to be incomplete when compared to the web interface or the browser plugin. I am one of the admin users for my company. The lpass tool version I used for testing were 1.0.0 from debian unstable, and 1.2.1 from https://github.com/lastpass/lastpass-cli. For example, when I do something like:lpass ls | grep -i -c something I find only 9 entries while the web interface search has many more (49 in my example). I have logged out and back in, removed all local configuration (.local.share/lpass) and upgraded the local software. This has made no difference.

So, in short, the problem is: there's a difference between the webinterface and the lpass tool. Can you help me find out where this comes from?

[edit] To clarify, this is not because the webinterface also searches in fields other than the name. There really are items missing that should be there. I forgot to mention that.

**Comment by adrighem** [comment]
Hmmm, you can close this issue. Apparently, lpass doesn't show items at all where I can't view the password.
